library/kani/src/slice.rs

```diff
@@ -1,8 +1,6 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-use crate::{any, assume, Arbitrary};
-use std::alloc::{alloc, dealloc, Layout};
-use std::ops::{Deref, DerefMut};
+use crate::{any, assume};
```

Comment:
Are there any users relying on `any_slice_of_array`, `any_slice_of_array_mut`, or `any_range`? Should we deprecate these functions as well?

Reply:
I'm not sure if there are any users that use them, but these functions do not suffer from the same issues that `any_slice` had (being too conservative at a severe expense of performance), and they might be convenient to use. So I prefer if we keep them.